mqlc/mqlc.go

```diff
@@ -1710,25 +1782,25 @@ func (c *compiler) expandListResource(chunk *llx.Chunk, ref uint64) (*llx.Chunk,
 	return newChunk, newType, newRef
 }
 
-func (c *compiler) expandResourceFields(chunk *llx.Chunk, typ types.Type, ref uint64) {
+func (c *compiler) expandResourceFields(chunk *llx.Chunk, typ types.Type, ref uint64) error {
```

Comment:
from what I can tell, it doesn't look like we care about error anywhere. It might be better to return a bool indicating success

Reply:
Fixed. Thanks for the suggestion.